pydocstringformatter/formatting/__init__.py

```diff
@@ -1,11 +1,14 @@
 __all__ = ["FORMATTERS"]
 
+from typing import List
+
+from pydocstringformatter.formatting.base import Formatter
 from pydocstringformatter.formatting.formatter import (
     BeginningQuotesFormatter,
     ClosingQuotesFormatter,
 )
 
-FORMATTERS = [
+FORMATTERS: List[Formatter] = [
```

Comment:
```suggestion
FORMATTERS = [
```

Reply:
Right now it's all StringFormatter so mypy infer it as a List of StringFormatter and it's a problem in `_register_arguments_formatters`. We might want to be excplicit for this one.

Reply:
Ah, understood! Never mind then.